src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc

```diff
@@ -172,6 +179,17 @@ To change the access mode for workspace PVCs, pass the `CHE_INFRA_KUBERNETES_PVC
 
 Another restriction is that only pods in the same namespace can use the same PVC.  The `CHE_INFRA_KUBERNETES_PROJECT` environment variable should not be empty.  It should be either the Che server namespace where objects can be created with the Che service account (SA) or a dedicated namespace where a token or a username and password need to be used.
 
+[id="per-workspace-pvc-strategy"]
+=== PerWorkspace PVC Strategy
+
+To define the unique strategy, set `CHE_INFRA_KUBERNETES_PVC_STRATEGY` to `per-workspace`.
+
+[id="how-the-per-workspace-pvc-strategy-works"]
+==== How the PerWorkspace PVC strategy works
+
+Basically it works in the same ways as Common PVC Strategy.
+The only different that all workspace volumes use the same PVC to store data (projects and workspace logs by default and whatever additional link:volumes.html[volumes] that a user can define.) but not all workspaces
```

Comment:
```suggestion
The only difference is that all workspace volumes use the same PVC to store data (projects and workspace logs by default and any additional link:volumes.html[volumes] that a user can define) but not all workspaces
```
Could you please explain what the end of the sentence means ? (`but not all workspaces`) What is it in relation to?

Reply:
I wanted to make clear different between `Per workspace` and `Common` strategies
So, The only difference is that **all workspace volumes** use the same PVC ... but **not all workspaces**.
Agree that it looks confusing. Should I just remove `but not all workspaces` or would you help me rephrase it?